verifier/src/x509/certs.rs

```diff
@@ -123,16 +138,35 @@ impl<'a> TryFrom<&'a [u8]> for UnverifiedCertificate<'a> {
 
 #[cfg(test)]
 mod test {
-    extern crate alloc;
-
     use super::*;
+    use alloc::string::ToString;
     use const_oid::ObjectIdentifier;
     use yare::parameterized;
 
     const LEAF_CERT: &str = include_str!("../../data/tests/leaf_cert.pem");
     const PROCESSOR_CA: &str = include_str!("../../data/tests/processor_ca.pem");
     const ROOT_CA: &str = include_str!("../../data/tests/root_ca.pem");
 
+    #[parameterized(
+        root = { ROOT_CA },
+        processor = { PROCESSOR_CA },
+        leaf = { LEAF_CERT },
+    )]
```

Comment:
Why parametrize these values if there's only one value for each param? Consider removing this usage if there isn't a good reason...